test/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ include("Boost.cmake")
 GetBoost()
 
 include("lp_solver.cmake")
-Get_LP_solver()
+Get_LP_Solver()
 
 find_library(LP_SOLVE NAMES liblpsolve55.so liblpsolve55.dylib PATHS /usr/lib/lp_solve /opt/local/lib)
```

Comment:
You have not removed this line which should be removed. Maybe that's why they are failiing on the the server tests and getting built correctly on your machine.